scripts/sortTermsAlphabetically.gradle

```diff
@@ -0,0 +1,108 @@
+import static groovy.io.FileType.FILES
+import java.util.regex.Matcher
+
+class Term {
+    String term
+    String captionEN
+    String captionDE
+    String includeFragment
+
+    Term(String term, String captionEN, String captionDE, includeFragment) {
+        this.term = term
+        this.captionEN = captionEN
+        this.captionDE = captionDE
+        this.includeFragment = includeFragment
+    }
+}
+
+task sortTermsAlphabetically(
+        description: 'collect terms and create an alphabetically orders structure',
+        group: 'isaqb-glossary'
+) {
+    doLast {
+        def terms = createTermsContent()
+
+        sortTermsEN(terms)
```

Comment:
Ok, now this is great. If we create the files for both German _and_ English, we can even remove more code from the 0-structure file and only include the German and English structure files. Then we can make this the first job that is _always_ executed.

Reply:
Ok.